patch/ros-noetic-rviz.patch

```diff
@@ -15,8 +15,8 @@ index 674e1033..7deeb26d 100644
 +find_package(assimp REQUIRED)
  
  find_package(OGRE QUIET COMPONENTS Overlay)
- if(NOT OGRE_FOUND)
-@@ -97,10 +92,10 @@ if(APPLE)
+ if(OGRE_FOUND)
```

Comment:
just a qucik question if you have any insight: it looks like this condition is flipped? I can't see the context here, but just wanted to check if you saw this, too?

Reply:
I wonder if that's what was making the patch fail because this doesn't appear be a change applied by the patch but context around a change.

Reply:
yep, indeed, this changed upstream, I just checked. All good!